### PR TITLE
Issue #616

### PR DESCRIPTION
**Pull request by stepanzharychevbroadcom**

### CODE REVIEW

File: src/ZoweUSSNode.ts [hunk 1]
```diff
@@ -278,14 +278,21 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
...
-        vscode.commands.executeCommand("zowe.uss.refreshUSSInTree", this);
-        vscode.commands.executeCommand("zowe.uss.ZoweUSSNode.open", this);
+        if (this.isFolder) {
+            await vscode.commands.executeCommand("zowe.uss.refreshAll");
+        } else {
+            await vscode.commands.executeCommand("zowe.uss.refreshUSSInTree", this);
+        }
+
+        if (!this.isFolder) {
+            await vscode.commands.executeCommand("zowe.uss.ZoweUSSNode.open", this);
```
Comment (lauren-li) [inline src/ZoweUSSNode.ts]: Does this line need to be in its own `if (!this.isFolder)` condition, or could it be part of the `else` block at line 289, after the `refreshUSSInTree` command?

**Comment by lauren-li** [comment]
With this branch, I no longer see the HTTP 500 error upon USS folder rename. I did have a question about the code (see my comment above). Thanks @stepanzharychevbroadcom for working on this!

File: src/ZoweUSSNode.ts (hunk 1 above)
Comment (stepanzharychevbroadcom) [reply src/ZoweUSSNode.ts]: I did it in this form to align with more advanced implementation in another PR.
https://github.com/zowe/vscode-extension-for-zowe/blob/af3719e4aff62fb7bd0fa470a0b3ca6470b1d35a/src/ZoweUSSNode.ts#L292

If it's a problem I can change it - no worries

Comment (stepanzharychevbroadcom) [reply src/ZoweUSSNode.ts]: @lauren-li Please, let me know if you have other concerns

Comment (lauren-li) [reply src/ZoweUSSNode.ts]: @stepanzharychevbroadcom To me, it makes more sense to have this line as part of the `else` block for now, and then later move it into its own `if (!this.isFolder) &&...` block in the PR for the more advanced implementation.

Comment (stepanzharychevbroadcom) [reply src/ZoweUSSNode.ts]: @lauren-li Changed it.

Comment (lauren-li) [reply src/ZoweUSSNode.ts]: Thanks @stepanzharychevbroadcom !

**Comment by lauren-li** [comment]
Since this PR is a fix, should we change it to be merging into the v1.3.x branch rather than master?

**Pull request closed without merging**